autokeras/blocks/wrapper.py

```diff
@@ -211,3 +218,423 @@ class GeneralBlock(block_module.Block):
 
     def build(self, hp, inputs=None):
         raise NotImplementedError
+
+
+class SegmentationBlock(block_module.Block):
+    """Block for image semantic segmentation.
+
+    The image block is a block choosing from ResNetBlock, XceptionBlock, ConvBlock,
+    which is controlled by a hyperparameter, 'block_type' from the paper
+    https://arxiv.org/pdf/1606.00915.pdf.
+
+    This image block is the task of semantic segmentation by applying g the
+    ‘atrous convolution’ with upsampled filters for dense feature extraction.
+    Then further extend it to atrous spatial pyramid pooling, which encodes
+    objects as well as image context at multiple scales.
+    To produce semantically accurate predictions and detailed segmentation maps
+    along object boundaries, we also combine ideas from deep
+    convolutional neural networks and fully-connected conditional random fields.
+
+    # Arguments
+        block_type: String. 'resnet', 'xception', 'vanilla'. The type of Block
+            to use. If unspecified, it will be tuned automatically.
+        classes: Int. Number of desired classes. If classes != 21,
+                last layer is initialized randomly.
```

Comment:
incorrect indent. It should be 4 spaces indent comparing to the line above. Please correct other such indents as well.